docs/content/packages/web-api.md

```diff
@@ -131,6 +131,20 @@ const conversationId = '...';
   });
 })();
 ```
+
+A dynamic method name can either be in the format shown above, to be sent with the [`slackApiUrl`](#custom-api-url)
+option, or as an absolute URL. Setting the
+[`allowAbsoluteUrls`](/reference/web-api/interfaces/WebClientOptions#allowabsoluteurls)
+option to `false` sends all requests to the `slackApiUrl` option value:
+
+```javascript
+const { WebClient } = require('@slack/web-api');
+
+const web = new WebClient(token, {
+  allowAbsoluteUrls: false,
+});
+```
```

Comment:
Open to suggestions as always, but wanted to make sure a note of this option is included in these details 📚 ✨

<img width="847" alt="docs" src="https://github.com/user-attachments/assets/c4091215-bc03-4507-9a4e-abd130a03759" />

Reply:
🔗 And a link for reference: https://tools.slack.dev/node-slack-sdk/web-api#call-a-method